pom.xml

```diff
@@ -319,6 +319,21 @@
 			<artifactId>powermock-api-mockito2</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.thymeleaf</groupId>
+			<artifactId>thymeleaf</artifactId>
+			<version>3.0.11.RELEASE</version>
+		</dependency>
+		<dependency>
+			<groupId>commons-io</groupId>
+			<artifactId>commons-io</artifactId>
+			<version>2.6</version>
+		</dependency>
+		<dependency>
+			<groupId>com.github.ben-manes.caffeine</groupId>
+			<artifactId>caffeine</artifactId>
+			<version>2.8.2</version>
+		</dependency>
```

Comment:
So the set of dependencies here should be kept as minimal as possible. These should be moved to the `dependencyManagement` section and then included in the POM where they are necessary (possibly only in the OMOD). I'd also drop the dependency on commons-io or change it's scope to "provided", since it's included in the platform.